code/modules/awaymissions/maploader/reader.dm

```diff
@@ -470,12 +465,12 @@ GLOBAL_DATUM_INIT(_preloader, /dmm_suite/preloader, new())
 // so that one can have separate "unpowered" areas for ruins or whatever,
 // yet have a single area type for use of mapping, instead of creating
 // a new area type for each new ruin
-/dmm_suite/loaded_map
+/datum/dmm_suite/loaded_map
 	parent_type = /datum
```

Comment:
This isnt needed anymore

Reply:
You mean the `parent_type` right?

Reply:
Yeah. You don’t need to set `parent_type` to datum when it’s already a datum.

Reply:
Alright, removed